docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,98 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
+
+Rationale
+---------
+
+There is often the case where a set of documentation has been built,
+and it is kept online but grows outdated over time.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in Google,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow project maintainers to request the *delisting* of forks that are old and outdated.
+
+Specification
+-------------
+The use cases covered by this policy are:
+
+Outdated docs
+    Documentation that is not being actively maintained or updated.
+
+Unofficial forks
+    Forks of a project documentation that are not linked to from the project in any way.
+
+Implementation
+--------------
+
+Unofficial projects
+~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unofficial* when ALL of the following are met:
+
+* It is not hosted on an official domain of the project
+* It is not linked from the project's README, description, or source code
+
+Unmaintained projects
+~~~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unmaintained* when ALL of the following are met:
+
+* The project is only serving content from releases and commits of a project over 6 months old
+
+Reachability
+~~~~~~~~~~~~
+
+for matters concerning projects that the user owns.
+In every case where contacting the user is necessary,
+the core team will reach out using one of the following means of contact:
+
+* E-mail address on file in the user's profile
+* E-mail address on the GitHub user account where the project is pointed
+
+The core team will stop trying to reach the user after 2 weeks and the user
+will be considered *unreachable*.
+
+Delisting of an abandoned project
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Projects that are determined to be *Unofficial* and *Unmaintained* will have a ``robots.txt`` file added that removes them from all search results.
```

Comment:
Maybe we should put the content of the robots.txt here. And looks like we would need to implement that before releasing this policy.

Reply:
This is now added, although I'm unsure about how we actually add this file as admins. We can talk about that later, but please have a look at the suggestion, it's pretty basic :)